plugins/quota/actor.go

```diff
@@ -0,0 +1,105 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	gderrors "github.com/gluster/glusterd2/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+// IsQuotadAffectedNode returns true if there are local bricks of volume on
+// which quota is enabled
+func IsQuotadAffectedNode() bool {
+	volumes, e := volume.GetVolumes()
+	if e != nil {
+		log.WithError(e).Error("Failed to get volumes")
```

Comment:
Pass a logger to this function and use that to log.

Reply:
done